docs/connectors/rhoas-cli-getting-started-connectors/README.adoc

```diff
@@ -77,131 +78,322 @@ END GENERATED ATTRIBUTES
+* You have a {product-long-kafka} instance running and have a topic called `test-topic`.
+* You have a service account created that has read and write access to the Kafka topic and you know the credentials (ID and secret).
 
+.Procedure
+
+. Create a source Connectors instance by specify the source connector's configuration file. For example, the data generator configuration file is `test-generator.json`.
++
+[source,subs="+quotes"]
+----
+$ rhoas connector create --file=test-generator.json 
+----
 +
-Enter `my-connector` as the connector's name.
+You're prompted to provide details for the Connectors instance.
+
+.. For *Set the Connectors namespace*, select your namespace from the list. For example, select `eval-namespace`.
+
+.. For *Service Account Client ID*, type or paste your ID.
+
+.. For *Service Account Client Secret*, type or paste your secret.
++ 
+A message states "Successfully created the Connectors instance".
+
+.. Wait until the status of the Connectors instance is *Ready*. 
 +
-Select the namespace you created when prompted.
+To check the status:
 +
-If prompted select the kafka instance you created with the topic you want to send messages too.
+[source,subs="+quotes"]
+----
+$ rhoas connector list
+----
+
+.. Verify that the your source Connectors instance is producing messages.
+
+. Create a sink Connectors instance by specify the sink connector's configuration file. For example, the HTTP sink configuration file is `test-http.json`.
 +
-Enter the Service Account Client Id when prompted.
+[source,subs="+quotes"]
+----
+$ rhoas connector create --file=test-http.json 
+----
 +
-Enter the Service Account Client Secret when prompted.
+You're prompted to provide details for the Connectors instance.
+
+.. For *Set the Connectors namespace*, select your namespace from the list. For example, select `eval-namespace`.
+
+.. For *Service Account Client ID*, type or paste your ID.
 
-. Once the {product-long-connectors} is running run the following command to see if the connector is producing messages as configured.
+.. For *Service Account Client Secret*, type or paste your secret.
 
+A message states "Successfully created the Connectors instance".
+
+. Wait until the status of the Connectors instance is *Ready*. 
++
+To check the status:
++
 [source,subs="+quotes"]
 ----
-$ rhoas kafka topic consume --name=my-topic --partition=0 --wait
+$ rhoas connector list
 ----
 
-.Verification
-* Is there a connectors instance running called `my-connector`.
-* Is there messages being received as expected? 
+. Verify that the your sink Connectors instance is receiving messages by viewing your link:https://webhook.site[Webhook.site^] page in a web browser.
+
 
-ifdef::qs[]
-[#conclusion]
-====
-Congratulations! You successfully completed the quick start for creating a connectors instance with the `rhoas` CLI.
-====
-endif::[]
+[role="_additional-resources"]
+.Additional resources
+* To access the `rhoas connectors` help page, type `rhoas connector -h`
```

Comment:
```suggestion
* To access the `rhoas connector` help page, type `rhoas connector -h`
```
It might be better to have a new section for "Commands for managing {product-connectors}", similar to the equivalent section in the "Getting started with the rhoas CLI" guides for Kafka and Service Registry? And include all of the different `rhoas` commands that are mentioned in this doc: 
* `rhoas connector -h`
* `rhoas kafka -h`
* `rhoas kafka acl -h`
* `rhoas kafka topic -h`
* `rhoas context -h`
* `rhoas service-account -h`

Not sure if it's best to list them alphabetically, or in the order in which they're used in the doc -- will leave that decision up to you!

Reply:
You've corrected the typo -- just unresolving the comment to track the other suggestion about creating a new section for the commands.